flyway/build.gradle

```diff
@@ -4,8 +4,10 @@ plugins {
 
 dependencies {
     annotationProcessor mn.micronaut.graal
+    annotationProcessor mnSerde.micronaut.serde.processor
 
     compileOnly libs.graal.svm
+    compileOnly mnSerde.micronaut.serde.api
```

Comment:
should not `mnSerde.micronaut.serde.api` be `api` instead of `compileOnly`?

Reply:
Wouldn't that mean that we pull the serde API library into someone's classpath if they use flyway? Even if they want jackson-databind?

Reply:
yes, but that will happen always if you want to expose an api and your API classes need serde api annotations.

Reply:
But people using this don't need Serde do they?  Isn't it a placeholder to generate classes in this module at compile-time?

[I was following what we do with micronaut-data](https://github.com/micronaut-projects/micronaut-data/blob/f15a90715b5e0c29ffcde67e47aa98b5f83b2b17/data-model/build.gradle)

Reply:
Another datapoint 😉 https://github.com/micronaut-projects/micronaut-liquibase/blob/master/liquibase/build.gradle